packages/editor-ui/src/composables/useCanvasOperations.ts

```diff
@@ -0,0 +1,215 @@
+import type { CanvasElement } from '@/types';
+import type { INodeUi, XYPosition } from '@/Interface';
+import { QUICKSTART_NOTE_NAME, STICKY_NODE_TYPE } from '@/constants';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { useHistoryStore } from '@/stores/history.store';
+import { useUIStore } from '@/stores/ui.store';
+import { useTelemetry } from '@/composables/useTelemetry';
+import { useExternalHooks } from '@/composables/useExternalHooks';
+import { MoveNodeCommand, RemoveConnectionCommand, RemoveNodeCommand } from '@/models/history';
+import type { Connection } from '@vue-flow/core';
+import { mapCanvasConnectionToLegacyConnection } from '@/utils/canvasUtilsV2';
+import type { IConnection } from 'n8n-workflow';
+
+export function useCanvasOperations() {
+	const workflowsStore = useWorkflowsStore();
+	const historyStore = useHistoryStore();
+	const uiStore = useUIStore();
+
+	const telemetry = useTelemetry();
+	const externalHooks = useExternalHooks();
+
+	/**
+	 * Node operations
+	 */
+
+	function updateNodePosition(
+		id: string,
+		position: CanvasElement['position'],
```

Comment:
Why not just `Position` from  `import type { Position } from '@vueuse/core';`? :P 
```suggestion
		position: Position,
```

Reply:
Updated.